dockernet/docker-compose.yml

```
@@ -173,6 +173,42 @@ services:
 
   # Fill in new host zone being tested here
 
+  umee1:
+    image: stridezone:umee
+    volumes:
+      - ./dockernet/state/umee1:/home/umee/.umeed
```

replace all these volumes mounts with:
```suggestion
      - ./dockernet/state/umee1:/home/umee/.umee
```

`.umee` instead of `.umeed`

---

Yes, the mounts were wrong !! Suggested is correct

```
/home/umee/.umee```